storage/backup/backup-cli/src/coordinators/replay_verify.rs

```diff
@@ -177,11 +178,13 @@ impl ReplayVerifyCoordinator {
             }
         }
 
-        let txn_manifests = transactions.into_iter().map(|b| b.manifest).collect();
         TransactionRestoreBatchController::new(
             global_opt,
             self.storage,
-            txn_manifests,
+            transactions
+                .into_iter()
+                .map(|t| t.manifest)
+                .collect::<Vec<_>>(),
```

Comment:
is it necessary?

Reply:
not necessary. found them equivalent so didn't change it back